heareval/tasks/config/nsynth_pitch.py

```diff
@@ -0,0 +1,24 @@
+"""
+Configuration for the nsynth pitch detection task
+"""
+
+from .dataset_config import *  # noqa: F403, F401
+
+TASKNAME = "nsynth-pitch-v2.3.3"
+
+TRAIN_DOWNLOAD_URL = (
+    "http://download.magenta.tensorflow.org/datasets/nsynth/nsynth-train.jsonwav.tar.gz"
+)
+VALIDATION_DOWNLOAD_URL = (
+    "http://download.magenta.tensorflow.org/datasets/nsynth/nsynth-valid.jsonwav.tar.gz"
+)
+TEST_DOWNLOAD_URL = (
+    "http://download.magenta.tensorflow.org/datasets/nsynth/nsynth-test.jsonwav.tar.gz"
+)
+
+SAMPLE_LENGTH_SECONDS = 4.0
+
+# Number files to include
+MAX_TRAIN_FILES = 8000
+MAX_VAL_FILES = 1000
+MAX_TEST_FILES = 1000
```

Comment:
Again we can decide on what we want the full number of samples for the dataset to be here. Full nsynth is quite large, 289,205 train, 12,678 valid, and 4096 test. Probably want to use the full test set, just filter out pitches outside piano range. What do you think about the size for train / valid? A similar size to google speech command in terms of duration would be a bit over 20k samples for training.

Reply:
@jorshi I would include these details as a comment, but by default just use the entire set (None)

Reply:
Sure sounds good. One thing to note is that running this on the entire NSynth dataset is going to take up alot of disk space. Between the workdir and task directory something like 700GB. So not sure how practical it will be for the average person to run this -- or for us to use that for the evaluation.